packages/suite/src/utils/suite/tor.ts

```diff
@@ -1,5 +1,6 @@
 import { parseHostname } from '@trezor/utils';
 import { TOR_URLS } from '@suite-constants/tor';
+import { TorStatus } from '@suite-reducers/suiteReducer';
```

Comment:
It's sometimes not a good idea to import `reducers` from `utils`, as when you'd like to use a util inside a reducer (which is imo very common case), you get a cyclic dependency. Just sharing my experience to consider.

Reply:
Moved to `/types` 0d594e7c0